pages/01.metakube/01.release-notes/default.en.md

```diff
@@ -170,7 +170,7 @@ page-toc:
 
 #### 2019-12-17
 
-* On OpenStack every Node now gets an additional label `machine-controller/host-id` that contains the id of the physical server the Node is scheduled on.
+* On OpenStack every Node now gets an additional label `machine-controller/physical-host-id` that contains the id of the physical server the Node is scheduled on.
```

Comment:
In the release notes it would probably be better to write that we updated the label instead of overwriting the old notes.
This way customers could see the change probably and update to the new one